src/declarations.d.ts

```diff
@@ -1,3 +1,5 @@
+type nullish = null | undefined | void;
```

Comment:
I think that is not good idea to mix `void` with `null` and `undefined`. If variable is `nullish` I expect some defined value or `null` or `undefined`